db/migrate/20200617183058_create_ahoy_visits_and_events.rb

```diff
@@ -0,0 +1,61 @@
+class CreateAhoyVisitsAndEvents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :ahoy_visits do |t|
+      t.string :visit_token
+      t.string :visitor_token
+
+      # the rest are recommended but optional
```

Comment:
I think we should remove every column we can't make an extremely compelling case for. Many of them go way past tracking how successful a campaign is and would prefer us to always take the "as little as possible" approach for storing user data.